### PR TITLE
QJB-175

### PR DESCRIPTION
**Pull request by osamaramihafez**
Addresses #175 

Files to review:
- src/services/twilio/sms.ts
- src/services/postgres/index.ts
- src/services/postgres/connect.ts
- src/server.ts
- src/index.ts
- script/run_tests.sh

Main Changes:
- Switch to using pg pool instead of client.
- Gracefully terminate server on manual exit.
- Notify admin (us) when postgres server is terminated

### CODE REVIEW

**Comment by osamaramihafez** [comment]
Some minor changes needed before merge:
- Ensure SMS is not sent during development or staging
- Clean up

File: src/index.ts
Comment (osamaramihafez) [inline src/index.ts:7]: We need to refactor code eventually, in some places we use camel case, in others we use pothole

File: src/services/twilio/sms.ts
Comment (osamaramihafez) [inline src/services/twilio/sms.ts:27]: @ibrahim681 let me know if you'd like to be added iA

Comment (ibrahim681) [reply src/services/twilio/sms.ts:27]: that would be great😊

File: src/index.ts
Comment (ibrahim681) [reply src/index.ts:7]: Ya, it is the same with file names. perhaps a consistent convention should be devised to avoid confusion

File: .env
Comment (ibrahim681) [inline .env:19]: I guess we put key storage on hold🥲

File: script/run_tests.sh
Comment (ibrahim681) [reply script/run_tests.sh:9]: pkill will send SIGTERM to all processes that match a given pattern. you could also use the process name instead of the pattern.

File: src/tests/api.test.ts
Comment (ibrahim681) [inline src/tests/api.test.ts:46]: what format styles are you using? when time permits we could set-up something like prettier so we are consistent

**Review by ibrahim681 — APPROVED** [review]
some general tips, nothing urgent. very cool😊

File: script/run_tests.sh
Comment (osamaramihafez) [reply script/run_tests.sh:9]: Tbh for the test I've decided it doesn't really matter atm. If we get to that problem later, we'll deal with it iA

File: src/tests/api.test.ts
Comment (osamaramihafez) [reply src/tests/api.test.ts:46]: Hmm, we are using prettier (see package.json). I'm not seeing a formatting problem here. Maybe it's github?

**Pull request merged**
